docs/dwarf-op.rst

```diff
@@ -17,10 +17,7 @@ random variation so each dwarf is unique.
 you can specify precisely what archtype you want a for given dwarf, and
 ``dwarf-op`` can generate a random dwarf that matches that archetype.
 
-Usage
------
-
-::
+Usage::
```

Comment:
Sorry, I didn't see this earlier. Could you revert this modification? Usage should be a header, not plain text.

Reply:
Oh, that probably happened during a merge conflict during rebase. Got a bit confused and must have done the wrong thing

Reply:
Yeah, the formatting is new. It took us a while to settle on this style. I must have changed it six or seven times.. that reminds me, I should look at Documentation.rst and make sure it reflects the latest standards